pom.xml

```diff
@@ -2139,6 +2139,15 @@
                 <vmtype>org.eclipse.jdt.internal.launching.macosx.MacOSXType</vmtype>
             </properties>
         </profile>
+        <profile>
+            <id>jdk8+</id>
+            <activation>
+                <jdk>[1.8,)</jdk>
+            </activation>
+            <properties>
+                <xDocLint>'-Xdoclint:none'</xDocLint>
```

Comment:
i don't think this is the correct way to do it. If tomorrow we want to support jdk9 you need to copy this very same thing into that profile too.

This should be a compiler argument. Check the 4.x branch, we already did this